client/blocks/eligibility-warnings/warning-list.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { map } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import ExternalLink from 'components/external-link';
+import SectionHeader from 'components/section-header';
+
+export const WarningList = ( {
+	translate,
+	warnings,
+} ) => {
+	return (
+		<div>
+			<SectionHeader label={ translate(
+				"By proceeding you'll lose %d feature:",
```

Comment:
Do we explain somewhere else _why_ they will lose these features? If not, maybe something like:

`In order for your site to support this plugin you'll lose %d feature:`